src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,241 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class CalculatorTest {
+    private static final double POSITIVE_NUMBER = 253.0093;
+    private static final double NEGATIVE_NUMBER = -137.057;
+    private static final double DELTA = 0.0001;
+    private static final double ZERO = 0;
+    private static final char PLUS = '+';
+    private static final char MINUS = '-';
+    private static final char MULTIPLICATION = '*';
+    private static final char DIVISION = '/';
+    private static final char POW = '^';
+    private static final char ILLEGAL_ARGUMENT = 'g';
+    private static Calculator calculator;
+    private double actual;
+    private double expected;
+
+    @BeforeAll
+    static void creatCalculator() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    void summationOfTwoPositiveNumber_Ok() {
```

Comment:
```suggestion
    void calculate_sumOfTwoPositiveNumbers_ok() {
```
the convention is `<methodUnderTest>_<state>_<expectedBehavior>`
check other methods as well